index.html

```diff
@@ -14,6 +15,79 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+  <div class="container">
+
+    <header>
+      <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+  
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
```

Comment:
Try putting everything inside of your container. It's why part of your nav is off-screen on smaller screens and may be why you still have a horizontal scroll bar on mobile view.